Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -361,16 +424,37 @@ void AuthenticatorCoordinator::discoverFromExternalSource(const Document& docume
         });
     }
 
-    auto callback = [weakThis = WeakPtr { *this }, promise = WTFMove(promise), abortSignal = WTFMove(requestOptions.signal), weakPage = WeakPtr { document.page() }] (AuthenticatorResponseData&& data, AuthenticatorAttachment attachment, ExceptionData&& exception) mutable {
+    auto& globalObject = *JSC::jsCast<JSDOMGlobalObject*>(document.globalObject());
+    auto requestJSON = toInspectorJSON(globalObject, *requestOptions.publicKey);
+
+    auto ceremonyId = Inspector::IdentifiersFactory::createIdentifier();
+    InspectorInstrumentation::frameDidStartWebAuthenticationOperation(*frame, ceremonyId, requestJSON.releaseNonNull(), initiatorStackTrace.copyRef());
+
+    auto callback = [weakThis = WeakPtr { *this }, promise = WTFMove(promise),
+        requestJSON = WTFMove(requestJSON),
+        abortSignal = WTFMove(requestOptions.signal), weakFrame = WeakPtr { frame }, weakDocument = WeakPtr { document }
+        , initiatorStackTrace = WTFMove(initiatorStackTrace), ceremonyId
+    ] (AuthenticatorResponseData&& data, AuthenticatorAttachment attachment, ExceptionData&& exception) mutable {
```

Comment:
Style: this is a really odd way of breaking up this line.  id recommend either putting each value onto it's own line or having them all just on the same line